browser/app/js/App.js

```diff
@@ -19,28 +19,12 @@ import { Route, Switch, Redirect } from "react-router-dom"
 import Browser from "./browser/Browser"
 import Login from "./browser/Login"
 import web from "./web"
-import { minioBrowserPrefix } from "./constants"
-
-const AuthorizedRoute = ({ component: Component, ...rest }) => (
-  <Route
-    {...rest}
-    render={props =>
-      web.LoggedIn() ? (
-        <Component {...props} />
-      ) : (
-        <Redirect to={`${minioBrowserPrefix}/login`} />
-      )
-    }
-  />
-)
```

Comment:
@kmkanagaraj as AuthorizedRoute is removed, will the user be redirected to login page if he is not logged in?

Reply:
@krishnasrinivas yes, it happens in BucketList.js

If the user has not loggedin and the bucket name is not in the url, user will be re-directed to login.